sdks/java/extensions/sql/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -160,7 +160,12 @@ SqlCreate SqlCreateTable(Span s, boolean replace) :
     <TABLE> ifNotExists = IfNotExistsOpt()
     id = CompoundIdentifier()
     fieldList = FieldListParens()
-    <TYPE> type = StringLiteral()
+    <TYPE>
+    (
+        type = StringLiteral()
+    |
+        type = SimpleIdentifier()
```

Comment:
Should we only keep `SimpleIdentifier()` ?

Reply:
If we only keep `SimpleIdentifier()`, this would be a breaking change to the synax.

Reply:
Right, `StringLiteral` is still the syntax for most cases.